packages/fxa-auth-server/lib/payments/capability.ts

```diff
@@ -80,11 +82,11 @@ export class CapabilityService {
    * Handle a Stripe Webhook subscription change.
    *
    * This handles broadcasting and refreshing the subscription capabilities for
-   * the product ids that were possibly updated.
+   * the price ids that were possibly updated.
    *
    * Stripe supports aligned subscriptions such that a single subscription can
    * include multiple items for multiple products. We iterate through this list
```

Comment:
This may perhaps be more future-proofing (as with multiple prices per subscription), but if a subscription can have multiple items for multiple products, then should this code still also iterate through products? I.e. is it possible the affected prices could be from different products? Would that make a difference here?

Reply:
I don't understand the question here.  But afaict the code is not making any assumption on whether the prices in a subscription is from one product or different products.

Reply:
It does iterate through all the products relevant for the plans.